dist/common/scripts/scylla_util.py

```diff
@@ -369,6 +394,14 @@ def __init__(self):
         self._type = self.__instance_metadata("instance-type")
         self.__populate_disks()
 
+    def is_aws_instance(self):
```

Comment:
why `is_gce_instance` is static, and this one isn't ?

Reply:
or at least a class method

Reply:
the whole scylla_util lacks basic architecture and basic usage of OOP design principles (Base class in this case). I am not going to redesign and rewrite this...
I can't use static method since I need `META_DATA_BASE_URL`

Reply:
I can use classmethod

Reply:
fixed